content/en/docs/faq.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Frequently asked questions
+type: docs
```

Comment:
Every page under `docs` will automatically be of type `docs`, you should not add an explicit qualifier.

```suggestion
linkTitle: FAQ
```